solution/ui/regulations/eregs-vite/src/components/JumpTo.vue

```diff
@@ -145,7 +143,10 @@ export default {
     methods: {
         async getParts(title) {
             const partsList = await getParts(title, this.apiUrl);
-            this.filteredParts = partsList.map((part) => part.name);
+            // temporarily filter part 75. See EREGCSC-1397
+            this.filteredParts = partsList
+                .map((part) => part.name)
+                .filter((part) => part !== "75");
```

Comment:
This all looks good and it well within the AC for this story.

However, we might want to slightly tweak the width for the Title select box at narrower widths, because with a title selected, it looks a little janky:
<img width="503" alt="Screen Shot 2023-05-17 at 11 51 08" src="https://github.com/Enterprise-CMCS/cmcs-eregulations/assets/1651907/4bb39ec1-d8cd-4967-b749-83aeba0ee023">

Reply:
I think it could be achieved by simply removing the media query mixin at line 88 of `_jump_to.scss`.

Reply:
Good catch! Thanks